### PR TITLE
Feature/speedoverview buttons endpoints

### PR DESCRIPTION
**Pull request by wyyi**

### CODE REVIEW

File: backend/app/Entities/Visit.php [hunk 1]
```diff
@@ -42,8 +44,6 @@ final class Visit extends Model
         'geo_position_id'
     ];
 
-    protected $with = ['session', 'page', 'visitor', 'geo_position'];
```
Comment (TumkoVolodymyr) [inline backend/app/Entities/Visit.php]: why do you need to delete this?

Comment (wyyi) [reply backend/app/Entities/Visit.php]: loads too much extra data when all i need are some fields on the visitor model

File: backend/app/Http/Controllers/Api/PageTimingController.php [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+        GetAverageTimingAction $getAveragePageLoadTimeAction
+    ) {
+        $this->getAveragePageLoadTimeAction = $getAveragePageLoadTimeAction;
+    }
+
+    public function getAveragePageLoading(FilterByPeriodHttpRequest $request): ApiResponse
+    {
+        $average = $this->getAveragePageLoadTimeAction->execute(
+            GetAverageTimingRequest::fromRequest($request),
+            'page_load_time'
```
Comment (lenchvolodymyr) [inline backend/app/Http/Controllers/Api/PageTimingController.php]: this prameter should be inside request. You can put request to variable and set it after all.

$averageRequest  = GetAverageTimingRequest::fromRequest($request);
$averageRequest->setParameter('page_load_time');

or event better to create from constructor

```
$averageRequest  = new GetAverageTimingRequest(
   $request->getStartDate(),
   $request->getEndDate(),
   'page_load_time'
);
```

File: backend/app/Actions/PageTimings/GetAverageTimingAction.php [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+    }
+
+    public function execute(ButtonDataRequest $request, $parameter): ButtonValue
+    {
+        $website_id = Auth::user()->website->id;
+        $period = $request->period();
+
+        $average = $this->repository
+            ->getAverageTiming($period, $website_id, $parameter);
+        return new ButtonValue(number_format($average));
```
Comment (lenchvolodymyr) [inline backend/app/Actions/PageTimings/GetAverageTimingAction.php]: do not format value on backend, it is responsibility of frontend

**Pull request merged**